src/pages/account/index.tsx

```diff
@@ -66,6 +66,5 @@ function Header() {
 function Separator() {
   return <div tw="hidden md:(inline font-semibold) ">·</div>;
 }
-AccountIndexPage.getLayout = (page: ReactNode) => {
-  return <AccountPageLayout>{page}</AccountPageLayout>;
-};
+
+(AccountIndexPage as NextPageWithLayout).getLayout = getAccountPageLayout;
```

Comment:
Don't think you have to do type assertion here, TypeScript doesn't complain, right?